spark/src/test/scala/io/ddf/spark/CacheDDFSuite.scala

```diff
@@ -0,0 +1,23 @@
+package io.ddf.spark
+
+
+class CacheDDFSuite extends ATestSuite {
+  createTableAirline()
+
+  test("cache a ddf") {
+    val ddf = manager.sql2ddf("select * from airline", "SparkSQL")
+    ddf.getRepresentationHandler.cache(false)
+    assert(ddf.getRepresentationHandler.isCached)
+
+    ddf.Transform.transformUDFWithNames(Array("newcol"), Array("arrdelay + depdelay"), Array("arrdelay", "depdelay"),
```

Comment:
Can we change to `ddfr = ddf.Transform...` and verify that both `ddfr` and `ddf` are cached? Similar for the test below.